.ci/test_r_package.sh

```diff
@@ -56,6 +56,7 @@ if [[ $OS_NAME == "linux" ]]; then
             texlive-latex-recommended \
             texlive-fonts-recommended \
             texlive-fonts-extra \
+            tidy \
```

Comment:
Fixes this NOTE from `R CMD check`:

```text
* checking HTML version of manual ... NOTE
Skipping checking HTML validation: no command 'tidy' found
```

Per "Writing R Extensions" ([link](https://cran.r-project.org/doc/manuals/R-exts.html))

> Optionally (including by `R CMD check --as-cran`) the HTML version of the manual is created and checked for compliance with the HTML5 standard. This requires a recent version[55](https://cran.r-project.org/doc/manuals/R-exts.html#FOOT55) of ‘HTML Tidy’, either on the path or at a location specified by environment variable `R_TIDYCMD`. Up-to-date versions can be installed from http://binaries.html-tidy.org/.